### PR TITLE
Update tests + user agent

### PR DESCRIPTION
**Pull request by jmelberg-okta**
- Updates the user agent to include device information
- Update the deployment target to iOS 9
- Fix rotated keys on test JWTs

### CODE REVIEW

File: OktaJWT/RequestsAPI.swift [hunk 1]
```diff
@@ -92,13 +92,19 @@ open class RequestsAPI: NSObject {
...
+        var responseData: Data?
+        let semaphore = DispatchSemaphore(value: 0)
+
+        let task = URLSession.shared.dataTask(with: request) { data, response, error in
+            responseData = data
+            semaphore.signal()
         }
+        task.resume()
+        semaphore.wait()
+        return responseData
```
Comment (jmelberg-okta) [inline OktaJWT/RequestsAPI.swift]: Our previous implementation was deprecated, requiring the update here.

File: Example/Tests/MiscTests.swift [hunk 2]
```diff
@@ -73,7 +73,7 @@ class MiscTests: XCTestCase {
 
     func testNoKIDInHeaderJWT() {
         let options = [
-            "issuer": TestUtils.issuer,
+            "issuer": "https://example.oktapreview.com/oauth2/default",
```
Comment (vijetmahabaleshwar-okta) [inline Example/Tests/MiscTests.swift]: Why can't we use `TestUtils.issuer` here?

Comment (jmelberg-okta) [reply Example/Tests/MiscTests.swift]: Originally I couldn't find a reliable source for creating an RS256 `alg` JWT. Found one and updated the test function.

File: Example/Tests/JWTTests.swift [hunk 3]
```diff
@@ -97,7 +97,7 @@ class JWTTests: XCTestCase {
         let options = [
             "issuer": TestUtils.issuer,
             "audience": TestUtils.audience,
-            "exp": true
+            "exp": true,
```
Comment (vijetmahabaleshwar-okta) [inline Example/Tests/JWTTests.swift]: Should we remove this trailing comma for consistency?

**Pull request merged**